internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/controller/TestHAASController.java

```diff
@@ -87,7 +87,7 @@ public void testStartHAASHelixControllerAsControllerClusterLeader() {
     }
   }
 
-  @Test(timeOut = 60 * Time.MS_PER_SECOND)
+  @Test(timeOut = 120 * Time.MS_PER_SECOND)
```

Comment:
Is this for mitigating flaky test?

Reply:
Yeah, I was getting flakiness on these two tests on my local, and increasing the timeout seemed to help.